quizzes/07_parameters.js

```diff
@@ -45,21 +45,42 @@ function restParams() {
     {city: 'San Diego', temp: 58.1},
   ]
   return weatherOnDemand(availableCities, 'Bali', 'London')
-  
+
   function weatherOnDemand() {
     // Use default parameters, argument destructuring
     // and other Array methods, to refactor this
     const allArgs = Array.prototype.slice.call(arguments)
     const weatherArray = allArgs[0]
     const cities = allArgs.slice(1)
-    
+
     return weatherArray.filter(cityData => {
       return cities.indexOf(cityData.city) !== -1
     })
   }
 }
 // log(restParams())
 
+// TODO: Make sure the function takes an object as input
+// This object, would ideally contain radius (r)
+// the x-coordinate (x), and y-coordinate(y)
+// Not all of these fields would always be provided
+// When not provided, r defaults to 1, while x and y defaults to 0
+function computeCircleArea(obj = {}) {
```

Comment:
Could you put this function (and its call) inside another function? That way we don't have to call this function with anything in particular when we do the `log` call. Also the container function should have a name that illustrates the feature we're trying to teach or demonstrate. Let me know whether that makes sense.